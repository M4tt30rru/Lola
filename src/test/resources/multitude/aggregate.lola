##Import "std.lola"
##Find aggregate ##Expression(_exp) s.t. ##Expression(_filter) with ##Expression(_id), ##Expression(_red);
 ##replace (##(_exp)).stream().filter(##(_filter)).reduce(##(_id), ##(_red));

aggregate f() s.t. x -> x % 2 == 0 with 0, (x, y) -> x + y;
