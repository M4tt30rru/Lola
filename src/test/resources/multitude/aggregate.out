(f()).stream().filter(x -> x % 2 == 0).reduce(0, (x, y) -> x + y);
